### PR TITLE
feat(tests): star routing runtime integration tests

### PR DESCRIPTION
**Pull request by jacobowitz**
Adds a bunch of integration tests, testing the new star routing runtimes.

Also doing a small change in the topology graph to send all messages on merging and not just the last.

### CODE REVIEW

File: tests/integration/runtimes/test_runtimes.py [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+
+
+@pytest.mark.asyncio
+# test gateway, head and worker runtime by creating them manually in the most simple configuration
+async def test_runtimes_trivial_topology():
+    worker_port = random_port()
+    head_port = random_port()
+    port_expose = random_port()
+    graph_description = '{"start-gateway": ["pod0"], "pod0": ["end-gateway"]}'
+    pod_addresses = f'{{"pod0": ["0.0.0.0:{head_port}"]}}'
```
Comment (JoanFM) [inline tests/integration/runtimes/test_runtimes.py:26]: is it a double dict?

File: tests/integration/runtimes/test_runtimes.py [hunk 2]
```diff
@@ -0,0 +1,492 @@
...
+    return worker_port, worker_process
+
+
+def _create_worker_runtime(port, name='', executor=None):
+    args = set_pea_parser().parse_args([])
+    args.port_in = port
+    args.name = name
+    if executor:
+        args.uses = executor
+    # args.polling = PollingType.ALL
```
Comment (JoanFM) [inline tests/integration/runtimes/test_runtimes.py:449]: let's remove this?

**Review by JoanFM — COMMENTED** [review]
small comment

File: tests/integration/runtimes/test_runtimes.py (hunk 1 above)
Comment (jacobowitz) [reply tests/integration/runtimes/test_runtimes.py:26]: no thats just escaping the curly bracket in the format string

**Review by JoanFM — APPROVED** [review]

**Pull request merged**